.github/workflows/deploy.yml

```
@@ -0,0 +1,211 @@
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.AWS_REGION }}
+
+    - name: Create deployment package
+      run: |
+        # Create a clean directory for the package
+        mkdir -p package
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)